lib/CL/devices/vulkan/pocl-vulkan.c

```diff
@@ -670,18 +652,19 @@ pocl_vulkan_init_device_ops (struct pocl_device_ops *ops)
   ops->init_queue = pocl_vulkan_init_queue;
   ops->free_queue = pocl_vulkan_free_queue;
 
-  // ########### IMAGES ############
-
-  //  ops->create_image = NULL;
-  //  ops->free_image = NULL;
-  //  ops->create_sampler = NULL;
-  //  ops->free_sampler = NULL;
-  //  ops->copy_image_rect = pocl_vulkan_copy_image_rect;
-  //  ops->write_image_rect = pocl_vulkan_write_image_rect;
-  //  ops->read_image_rect = pocl_vulkan_read_image_rect;
-  //  ops->map_image = pocl_vulkan_map_image;
-  //  ops->unmap_image = pocl_vulkan_unmap_image;
-  //  ops->fill_image = pocl_vulkan_fill_image;
+  /* ########### IMAGES ############ */
+  /*
+  ops->create_image = NULL;
```

Comment:
Can you call an abort unimplemented function for these too? Or does it report unsupported images with NULL pointers here?

Reply:
The image support is a flag in cl_device_id, and the clEnqueue* API checks that flag, so it's not necessary.